packages/design-system/src/styles/theme/oc-label.scss

```diff
@@ -0,0 +1,4 @@
+.oc-label {
+  margin-bottom: 0.4em;
```

Comment:
This will always add a margin, even if there is no label (which is rarely the case, but still).

<img width="374" alt="image" src="https://user-images.githubusercontent.com/50302941/223952630-e287d835-bca1-43ea-a0f1-a467b211a86d.png">

Reply:
change to `label.oc-label` ?

Reply:
Ahh sorry, it happens because of the `inline-block`, not because of the margin. Hmm... `.oc-label:not(:empty)` should work I believe.

Reply:
hm why is even a label used when it's empty?

Reply:
The element has to exist in the DOM for accessibility reasons, even when there is no text.